doc/overviews/rate-limiting.md

```diff
@@ -241,26 +249,27 @@ The selectors within the `when` conditions of a RateLimitPolicy are a subset of
 ### Examples
 
 Check out the following user guides for examples of rate limiting services with Kuadrant:
-* [Simple Rate Limiting for Application Developers](user-guides/simple-rl-for-app-developers.md)
-* [Authenticated Rate Limiting for Application Developers](user-guides/authenticated-rl-for-app-developers.md)
-* [Gateway Rate Limiting for Cluster Operators](user-guides/gateway-rl-for-cluster-operators.md)
-* [Authenticated Rate Limiting with JWTs and Kubernetes RBAC](user-guides/authenticated-rl-with-jwt-and-k8s-authnz.md)
+
+- [Simple Rate Limiting for Applications](../user-guides/ratelimiting/simple-rl-for-app.md)
+- [Authenticated Rate Limiting for Application](../user-guides/ratelimiting/authenticated-rl-for-app.md)
+- [Gateway Rate Limiting for Cluster Operators](../user-guides/ratelimiting/multi-rlp-multi-listener.md)
+- [Authenticated Rate Limiting with JWTs and Kubernetes RBAC](../user-guides/ratelimiting/authenticated-rl-with-jwt-and-k8s-authnz.md)
 
 ### Known limitations
 
-* RateLimitPolicies can only target HTTPRoutes/Gateways defined within the same namespace of the RateLimitPolicy.
-* 2+ RateLimitPolicies cannot target network resources that define/inherit the same exact hostname.
+- RateLimitPolicies can only target HTTPRoutes/Gateways defined within the same namespace of the RateLimitPolicy.
+- 2+ RateLimitPolicies cannot target network resources that define/inherit the same exact hostname.
 
 ## Implementation details
 
 Driven by limitations related to how Istio injects configuration in the filter chains of the ingress gateways, Kuadrant relies on Envoy's [Wasm Network](https://www.envoyproxy.io/docs/envoy/latest/configuration/listeners/network_filters/wasm_filter) filter in the data plane, to manage the integration with rate limiting service ("Limitador"), instead of the [Rate Limit](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_filters/rate_limit_filter) filter.
 
-**Motivation:** _Multiple rate limit domains_<br/>
+**Motivation:** _Multiple rate limit domains_
```

Comment:
we want a new line here... maybe one empty line in Markdown to have one line in render view?